### PR TITLE
Login backend

### PR DESCRIPTION
**Pull request by TLMatthies**
A lot of little things were changed, because I needed to remove logging so that I could see what my logs were doing. Main changes are in backend.js (authenticateUser), Login.jsx (submitForm), and MyApp.jsx (addAuthHeader, checkUserLogin). As of now, authenticateUser and addAuthHeader are untested. For how to use them, look to TA4 on canvas.

### CODE REVIEW

File: packages/react-frontend/src/Login.jsx
Comment (J-Dees) [inline packages/react-frontend/src/Login.jsx:1]: Something Tim and I discussed is the method of creating a user. With this new method of authentication, we must return the created user to the login page so they can get a JWT token. @TLMatthies: either I can implement this in my pull request or you can fix it in this one.

**Comment by J-Dees** [comment]
# Issue #70 Review:

- either switch create user to route the user to log in page or let me know so I can implement that

Looks great otherwise!

**Review by J-Dees — APPROVED** [review]

File: packages/react-frontend/src/Login.jsx
Comment (eamyzzaj) [reply packages/react-frontend/src/Login.jsx:1]: Great job getting this done guys!

**Review by eamyzzaj — COMMENTED** [review]
Just double checking if this PR is still good to merge? I know there were some issues with some out of date stuff but that might have been on another PR

**Comment by TLMatthies** [comment]
> Just double checking if this PR is still good to merge? I know there were some issues with some out of date stuff but that might have been on another PR

This branch is out of date. Closing it right now.

**Pull request closed without merging**